torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3199,7 +3195,8 @@ at::Tensor XLANativeFunctions::new_empty_strided(
 }
 
 at::Tensor XLANativeFunctions::narrow_copy(const at::Tensor& self, int64_t dim,
-                                           int64_t start, int64_t length) {
+                                           c10::SymInt start,
```

Comment:
OoO, why did we also change `start` to `SymInt`. Do we have to switch all int arguments to `SymInt` for symint kernels?

Reply:
No. But start should be a SymInt as it is a "size-y" int. Notice that dim doesn't get changed.